public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -108,33 +111,69 @@ export class GraphiteDatasource extends DataSourceApi<GraphiteQuery, GraphiteOpt
     if (!result || !result.data) {
       return { data };
     }
+
     // Series are either at the root or under a node called 'series'
     const series = result.data.series || result.data;
+
     if (!_.isArray(series)) {
       throw { message: 'Missing series in result', data: result };
     }
 
     for (let i = 0; i < series.length; i++) {
       const s = series[i];
+
       for (let y = 0; y < s.datapoints.length; y++) {
         s.datapoints[y][1] *= 1000;
       }
+
       const frame = toDataFrame(s);
 
       // Metrictank metadata
       if (s.meta) {
         frame.meta = {
           custom: {
-            request: result.data.meta, // info for the whole request
-            info: s.meta, // Array of metadata
+            requestMetaList: result.data.meta, // info for the whole request
+            seriesMetaList: s.meta, // Array of metadata
           },
         };
+
+        const rollupNotice = getRollupNotice(s.meta);
+        const runtimeNotice = getRuntimeConsolidationNotice(s.meta);
+
+        if (rollupNotice) {
+          frame.meta.notices = [rollupNotice];
+        } else if (runtimeNotice) {
+          frame.meta.notices = [runtimeNotice];
+        }
```

Comment:
right now rollup notice will take precedence but can check for the combination and have special notice text in future